README.md

```
@@ -1 +1,101 @@
-# signal-mitm
\ No newline at end of file
+# Minion in the Middle
+
+Welcome! This is an open source framework for security researchers, facilitating conducting man-in-the-middle attacks against end-to-end encrypted messaging applications.
+This is a research framework! We thus do not include any kind of obfuscation measures for attacks and will not accept contributions that attempt to add them.
+
+While this repository currently focuses on the open-source Signal Messenger, we hope to extend it to more encrypted messenger applications in the future.
+
+### Table of Contents
```

Markdownlint (0.35.0)

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)